imsim/ccd.py

```diff
@@ -34,6 +34,7 @@ def setup(self, config, base, file_num, logger):
         # Figure out the detector name for the file name.
         detnum = galsim.config.ParseValue(config, 'det_num', base, int)[0]
         camera = get_camera(config['camera'])
+        base['camera'] = config['camera']
```

Comment:
I don't think this is how we want to do this.  We have several places in the config that need the camera, and we always use `camera: "@output.camera"` to refer to the one in `output`.  We should do that here too where the image type has its own `camera` item.  Then if the user wants to replace any of these with something else (e.g. not use our output type, but keep the LSST_Image image type) there is a place to specify the `camera` item properly.